core/src/main/java/org/jdbi/v3/ArgumentRegistry.java

```diff
@@ -20,6 +20,7 @@
 import java.util.Optional;
 import java.util.concurrent.CopyOnWriteArrayList;
 
+import org.jdbi.v3.argument.BuiltInArgumentFactory;
 import org.jdbi.v3.tweak.Argument;
 import org.jdbi.v3.tweak.ArgumentFactory;
```

Comment:
`Argument` and `ArgumentFactory` want to move too?

I'd actually like to see the `tweak` package broken out and a lot of classes in `org.jdbi.v3` moved into subpackages by feature.